tests/scripts/arp_responder.py

```diff
@@ -5,12 +5,17 @@
 from collections import defaultdict
 import logging
 import scapy.all as scapy
+import ipaddress
+scapy.MTU = 1280
```

Comment:
Why is this change needed? What's the default and why is that problematic?

Reply:
This was an _attempt_ to replicate the behavior of the `-s` flag in `dumpcap`/`tcpdump` and how the space taken up by each packet in the RX ring changes. However, this doesn't appear to actually work, so it can be removed.

The default is 65535 bytes. By specifying a smaller snapshot size, assuming I'm reading the kernel documentation correctly, more packets can fit in the RX ring for a given buffer size. This can mean higher throughput of packets and/or higher ability to handle bursts of packets.